erpnext/setup/doctype/email_digest/email_digest.py

```diff
@@ -24,36 +24,17 @@ def __init__(self, *args, **kwargs):
 		self._accounts = {}
 		self.currency = frappe.db.get_value('Company',  self.company,  "default_currency")
 
-	def get_users(self):
-		"""get list of users"""
-		user_list = frappe.db.sql("""
-			select name, enabled from tabUser
-			where name not in ({})
-			and user_type != "Website User"
-			order by enabled desc, name asc""".format(", ".join(["%s"]*len(STANDARD_USERS))), STANDARD_USERS, as_dict=1)
-
-		if self.recipient_list:
-			recipient_list = self.recipient_list.split("\n")
-		else:
-			recipient_list = []
-		for p in user_list:
-			p["checked"] = p["name"] in recipient_list and 1 or 0
-
-		frappe.response['user_list'] = user_list
-
 	def send(self):
 		# send email only to enabled users
 		valid_users = [p[0] for p in frappe.db.sql("""select name from `tabUser`
 			where enabled=1""")]
-		recipients = list(filter(lambda r: r in valid_users,
-			self.recipient_list.split("\n")))
 
-		if recipients:
-			for user_id in recipients:
+		if self.recipients:
+			for recipient in self.recipients:
```

Comment:
```suggestion
			for row in self.recipients:
```
I think rohit suggested this in the last review too. Keep variable names clean. To avoid things like `recipient.recipient`. It's confusing as to what `recipient.recipient` means.
`row.recipient` or `entry.recipient` is much cleaner and tells me that I'm iterating over a child table and accessing an attribute.